examples/data.go

```diff
@@ -6,42 +6,48 @@ import (
 	ep "github.com/wrossmorrow/envoy-extproc-sdk-go"
 )
 
-type dataRequestProcessor struct{}
+type dataRequestProcessor struct {
+	opts *ep.ProcessingOptions
+}
 
-func (s dataRequestProcessor) GetName() string {
+func (s *dataRequestProcessor) GetName() string {
 	return "data"
 }
 
-func (s dataRequestProcessor) GetOptions() *ep.ProcessingOptions {
-	opts := ep.NewOptions()
-	opts.UpdateExtProcHeader = true
-	opts.UpdateDurationHeader = true
-	return opts
+func (s *dataRequestProcessor) GetOptions() *ep.ProcessingOptions {
+	return s.opts
 }
 
-func (s dataRequestProcessor) ProcessRequestHeaders(ctx *ep.RequestContext, headers map[string][]string) error {
+func (s *dataRequestProcessor) ProcessRequestHeaders(ctx *ep.RequestContext, headers map[string][]string, headerRawValues map[string][]byte) error {
```

Comment:
I think this is a place where passing a struct with string and byte fields is preferable; I'll make a branch with specific suggestions to work this idea out.

Reply:
with `*`, is to make lint happy